lms/resources/_js_config/__init__.py

```diff
@@ -381,9 +373,9 @@ def enable_file_picker_mode(  # noqa: PLR0913
                     "formAction": form_action,
                     "formFields": form_fields,
                     "promptForTitle": prompt_for_title,
-                    "autoGradingEnabled": self._application_instance.settings.get(
-                        "hypothesis", "auto_grading_enabled", False
-                    ),
+                    # Enable auto grading every where except in Sakai
```

Comment:
This controls only the "config side". But if you can't configure them to gbe auto gradable then we don't have to worry about anything on the dashboard side.